runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -374,6 +375,16 @@ private void handleTagTab(ScriptEvent event)
 				Toolkit.getDefaultToolkit().getSystemClipboard().setContents(stringSelection, null);
 				notifier.notify("Tag tab " + tagTab.getTag() + " has been copied to your clipboard!");
 				break;
+			case Tab.RENAME_TAB:
+				String renameTarget = Text.standardize(event.getOpbase());
+				chatboxPanelManager.openTextMenuInput("Rename " + renameTarget + "?")
```

Comment:
Why ask the user to confirm the option? They can just cancel out of the input box.

Reply:
Good point, I'll remove the confirmation.